### ISSUE TITLE
Uploading files from refile doesn't work

### ISSUE THREAD

**Issue by chamini2** [sferik/rails_admin#2435]
When uploading files with refile, support added in a recent commit, instead of the file's ID, it stores the string representation of the upload method:

    :005 > rank.award_id
     => "#<ActionDispatch::Http::UploadedFile:0x007fee4986bc28>"

Where `award_id` should be the refile ID.

This raises an exception when trying to view said file

    :006 > rank.award
     Refile::InvalidID: Refile::InvalidID

**Comment by mshibuya** [comment]
How do you configure your app to use refile?
Can you paste your model and RailsAdmin config?

**Comment by chamini2** [comment]
Well, I've made some changes since I wrote this issues, and trying everything out for submitting the info you requested, it did work, I guess I fixed some error without knowing.

But I think the problem was that I used:

    field :award_id, :refile

And removing the `_id`

    field :award, :refile

It works.

The hint was that in the issue I showed that `award_id` returned an `ActionDispatcher::Http::UploadedFile`, not a `String` with the id.
